pacman.py

```diff
@@ -254,15 +253,59 @@ def is_aur(package):
         return False
 
 
-def pacman(flags, pkgs=[], eflgs=[], pacman_bin=__PACMAN_BIN):
+def contents_of(packages):
+    """Get list of files and directories that are owned by these packages"""
+    s = pacman("-Ql", packages)
+    if s["code"] != 0:
+        raise Exception("Failed to get contents: {0}".format(s["stderr"]))
+    ret = dict()
+    for x in s["stdout"].split('\n'):
+        space = x.find(' ')
+        package = x[:space+ 1]
+        file = x[space:]
+        if package not in ret:
+            ret[package] = []
+        ret[package].append(file)
+
+    if type(packages) == str:
+        return ret[packages] if packages in ret else None
```

Comment:
Returns a single file list if passed a single package as a string. If passed a list of packages (even if the list contains just one element), the result will be grouped by package name.